app/controllers/concerns/pubmed_search.rb

```diff
@@ -2,8 +2,8 @@ module PubmedSearch
   extend ActiveSupport::Concern
 
   def search_pubmed(search_term)
-    @search_terms = search_term.split.join("+")
-    search_url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&retmode=json&retmax=20&term='+@search_terms
+    @search_term = search_term.split.join("+")
+    search_url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&retmode=json&retmax=20&term='+@search_term
```

Comment:
You can't do `PubmedSearch.new` if it's a module

On the other hand, having to initialize something (`pubmed = PubmedSearch.new(stuff: here)`) kinda sucks